packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -304,6 +305,14 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     return null;
   }
 
+  private _onRenderDescription = (props: ITextFieldProps): JSX.Element | null => {
```

Comment:
>null [](start = 73, length = 4)

undefined

Reply:
@manishgarg1 I've updated both `_onRenderDescription` and `_onRenderLabel`.

A bit different than you suggested though. Rather than replacing `null` with `undefined`, I was able to remove null from the return type by moving the guard into the render method. 
(I wasn't sure for the motivation behind your suggestion, so please let me know if I missed the point.)

The reason I did it this way was because changing `null` to `undefined` on the return type resulted in the following error at the `onRenderLabel` invokation, which I don't yet understand.
```
[ts] Cannot invoke an expression whose type lacks a call signature. Type 'IRenderFunction<ITextFieldProps> | ((props: ITextFieldProps) => Element | undefined)' has no compatible call signatures.
const onRenderLabel: IRenderFunction<ITextFieldProps> | ((props: ITextFieldProps) => JSX.Element | undefined)
```

Reply:
@manishgarg1 On further consideration, I don't think what I did will work for us.  We'd want the provided onRenderDescription to be invoked even when description is absent.  I've reverted this last change. 

If you still think `null` should be changed to `undefined`, can you please help me understand the motivation? And suggest a course of action given the error that I encountered above?